docker/dlang

```diff
@@ -11,6 +11,7 @@ set -xeu
 dist="$(lsb_release -cs)"
 
 # Add extra dlang APT repositories from Sociomantic in Bintray
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
```

Comment:
Moving this out of the base image will make adding bintray repos more painful for the users of the base image.

Reply:
You also want to add `ebtree` repo here (and other repos that are moved out from tools?) You are still installing ebtree below but now it will fail.

Reply:
Nevermind, `ebtree` was in the `dlang` repo, so this will work for the ebtrees still there.

Reply:
Yeah, no repos in the base image because we need it to build the tools that we need to add to the dlang image.